monai/auto3dseg/utils.py

```diff
@@ -260,3 +264,58 @@ def verify_report_format(report: dict, report_format: dict):
                 return False
 
         return True
+
+
+def algo_export_to_pickle(algo, pkl_filename: str, algo_templates_dir: str) -> None:
+    """
+    Export the Algo object to pickle file
+
+    Args:
+        algo: Algo-like object
+        pkl_filename: name of the pickle file
+        algo_templates_dir: the algorithm script folder which is needed to instantiate the object
+    """
+    data = {"obj_bytes": pickle.dumps(algo), "algo_templates_dir": algo_templates_dir}
+    data_bytes = pickle.dumps(data)
+    with open(pkl_filename, "wb") as f_pi:
+        f_pi.write(data_bytes)
+
+
+def algo_import_from_pickle(pkl_filename, algo_templates_dir: Optional[str] = None) -> Any:
```

Comment:
Please add the typehint for `pkl_filename`.

Thanks.

Reply:
Done. Thanks!